### PR TITLE
Add an action to update baselines/fix lints

### LINKED ISSUE #37765: typescript-bot: baseline-accept and fix-lints
Lots of the time people send changes to the repo but forget to update their baselines or have a lint issue. It would be great if @typescript-bot could be pinged to fix these things with the commands

```
@typescript-bot baseline-accept

@typescript-bot fix-lints
```

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
Fixes #37765

The workflow will more or less look like this

image

### CODE REVIEW

File: .github/workflows/accept-baselines-fix-lints.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - uses: actions/checkout@v2
+    - name: Use node version 12
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+        registry-url: https://registry.npmjs.org/
+
+    - name: Configure Git, Run Tests, Update Baselines, Apply Fixes
+      run: |
+        git config user.email "ts_bot@rcavanaugh.com"
```
Comment (orta) [inline .github/workflows/accept-baselines-fix-lints.yml:20]: perfect, blame it on Ryan!

Comment (DanielRosenwasser) [reply .github/workflows/accept-baselines-fix-lints.yml:20]: Okay, side note, all of our actions do this. Two things

1. Do we have a Microsoft account we can use @RyanCavanaugh?
2. Is there a way to use the person who triggered the PR's email?

**Comment by sheetalkamat** [comment]
I am assuming we can do this on PR or a branch and would make life so much easier to just accept baselines or fix lint errors rather than having to do them locally after finding out from bigger run

**Comment by weswigham** [comment]
I don't think the action will work on PRs from forks in this form.

**Comment by sheetalkamat** [comment]
May be make it bot work then ?

File: .github/workflows/accept-baselines-fix-lints.yml (hunk 1 above)
Comment (andrewbranch) [reply .github/workflows/accept-baselines-fix-lints.yml]: 1. typescriptbot@microsoft.com
2. I believe `${{github.actor}}@users.noreply.github.com` should do the trick

**Comment by DanielRosenwasser** [comment]
I think this is useful enough for team PRs.

File: .github/workflows/accept-baselines-fix-lints.yml (hunk 1 above)
Comment (DanielRosenwasser) [reply .github/workflows/accept-baselines-fix-lints.yml]: > I believe `${{github.actor}}@users.noreply.github.com` should do the trick

I think if your account isn't old enough, you have to use something like a special user ID email.

https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address#about-commit-email-addresses

> If you created your GitHub account after July 18, 2017, your GitHub-provided no-reply email address is a seven-digit ID number and your username in the form of `ID+username@users.noreply.github.com`. If you created your GitHub account prior to July 18, 2017, your GitHub-provided no-reply email address is your username in the form of `username@users.noreply.github.com`. You can get an ID-based GitHub-provided no-reply email address by selecting (or deselecting and reselecting) Keep my email address private in your email settings.

**Pull request merged**